Systems/external-heat.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>  
+
+<!-- c182s external heater
```

Comment:
Could you add a copyright block (GPL 2+) here? You can find an example in some of the files in the Systems folder. Make sure to put Benedikt Hallinger as the author in the copyright line.